### PR TITLE
Team Value PRs

### PR DESCRIPTION
**Pull request by sandeepmehta2155**

### CODE REVIEW

**Review by kajol-rep — COMMENTED** [review]
Good work Sandeep ! 
I would like to suggest few things to improve your design:

1.  In home page a picture is missing from featured series.
2. In home page you can keep the chatbot component at the bottom.
3. In products page you can increase the width of the card and let there be some space around the image so that it will look good.
4. In wishlist page  please have a look at heart icon it flowing outside the border.
5. In cart page remove from cart button is flowing out of the box.
6. Give some color hover effects to the buttons. So it will look like a button.

**Comment by sandeepmehta2155** [comment]
> Good work Sandeep !
> I would like to suggest few things to improve your design:
> 
> 1. In home page a picture is missing from featured series.
> 2. In home page you can keep the chatbot component at the bottom.
> 3. In products page you can increase the width of the card and let there be some space around the image so that it will look good.
> 4. In wishlist page  please have a look at heart icon it flowing outside the border.
> 5. In cart page remove from cart button is flowing out of the box.
> 6. Give some color hover effects to the buttons. So it will look like a button.

1. Image is now working
2. Removed Chat Bot as it is of no use
3. Card width is increased and I wanted image to occupy 100% of card width
4. Issue rectified
5. Issue rectified
6. Hover effect added

**Review by hntejas — CHANGES_REQUESTED** [review]
Hi Sandeep, here are some of my functional/UI suggestions

1) The site is not responsive at desktop resolutions
2) Product filter is below the footer and can be moved to a right place 
3) Add to cart button is difficult to access on the products 
4) The header is not responsive in mobile resolution 
5) If the user is already logged in/ the login page can have the logout option 
6) It is recommended to have  cursor as point on action buttons only

**Comment by sandeepmehta2155** [comment]
> Hi Sandeep, here are some of my functional/UI suggestions
> 
> 1. The site is not responsive at desktop resolutions
> 2. Product filter is below the footer and can be moved to a right place
> 3. Add to cart button is difficult to access on the products
> 4. The header is not responsive in mobile resolution
> 5. If the user is already logged in/ the login page can have the logout option
> 6. It is recommended to have  cursor as point on action buttons only

1. Made responsive
2. Filter moved to product page
3. Issue resolved
4. Made Responsive
5. Need to work on persistence
6. Wrongly inherited due to component library

**Comment by hntejas** [comment]
> > Hi Sandeep, here are some of my functional/UI suggestions
> > 
> > 1. The site is not responsive at desktop resolutions
> > 2. Product filter is below the footer and can be moved to a right place
> > 3. Add to cart button is difficult to access on the products
> > 4. The header is not responsive in mobile resolution
> > 5. If the user is already logged in/ the login page can have the logout option
> > 6. It is recommended to have  cursor as point on action buttons only
> 
> 1. Made responsive
> 2. Filter moved to product page
> 3. Issue resolved
> 4. Made Responsive
> 5. Need to work on persistence
> 6. Wrongly inherited due to component library

Hi Sandeep, Thanks for making the fixes. 

here are few of the issues that I observed.
1) The cart line is overflowing into the Cart summary
2) The Product filter is now not visible 
3) Sorting and filter buttons in the mobile resolution does not respond to click 
4) Some images in home page are not loading 
5) The cursor is still pointer in all pages

**Comment by sandeepmehta2155** [comment]
> > > Hi Sandeep, here are some of my functional/UI suggestions
> > > 
> > > 1. The site is not responsive at desktop resolutions
> > > 2. Product filter is below the footer and can be moved to a right place
> > > 3. Add to cart button is difficult to access on the products
> > > 4. The header is not responsive in mobile resolution
> > > 5. If the user is already logged in/ the login page can have the logout option
> > > 6. It is recommended to have  cursor as point on action buttons only
> > 
> > 
> > 
> > 1. Made responsive
> > 2. Filter moved to product page
> > 3. Issue resolved
> > 4. Made Responsive
> > 5. Need to work on persistence
> > 6. Wrongly inherited due to component library
> 
> Hi Sandeep, Thanks for making the fixes.
> 
> here are few of the issues that I observed.
> 
> 1. The cart line is overflowing into the Cart summary
> 2. The Product filter is now not visible
> 3. Sorting and filter buttons in the mobile resolution does not respond to click
> 4. Some images in home page are not loading
> 5. The cursor is still pointer in all pages

There was mild issue in deployment, changes have been updated

**Review by kajol-rep — APPROVED** [review]
I have reviewed your changes. You are GTG ahead.

File: src/Products-Page/product-display-page.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function ProductDisplay() {
+  return <div></div>;
```
Comment (hntejas) [inline src/Products-Page/product-display-page.js:2]: unused components can be cleaned up

**Review by hntejas — COMMENTED** [review]
Hi Sandeep, 

Here are some of my suggestions regarding clean code and code structuring practices. These are only for code organization and code readability and will not affect performance of the application

1) Components and Pages can be broken down and organized into different folders, the same with context and it's related utilities
2) For the naming conventions :
  a) components and it's files can be PascalCase
  b) variables and functions can be camelCase
  c) style related naming can be kebab-case
3) The file extensions can be maintained consistent as jsx or js.

File: src/Products-Page/product-display-page.js (hunk 1 above)
Comment (sandeepmehta2155) [reply src/Products-Page/product-display-page.js]: Used components removed

File: src/Cart-Page/cart.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
+import { Link } from "react-router-dom";
+import { useProd } from "../Products-Page/product-context";
+
+export const Cart = () => {
+  const { itemsInProduct, productDispatch } = useProd();
```
[the remark this answers is not in the document]
Comment (sandeepmehta2155) [reply src/Cart-Page/cart.js:5]: As I had one day to change the whole UI I wasn't aware of this thanks for pointing out. Awaiting for further review if any changes are to be made.

**Comment by hntejas** [comment]
> > Hi Sandeep,
> > Here are some of my suggestions regarding clean code and code structuring practices. These are only for code organization and code readability and will not affect performance of the application
> > 
> > 1. Components and Pages can be broken down and organized into different folders, the same with context and it's related utilities
> > 2. For the naming conventions :
> >    a) components and it's files can be PascalCase
> >    b) variables and functions can be camelCase
> >    c) style related naming can be kebab-case
> > 3. The file extensions can be maintained consistent as jsx or js.
> 
> I will surely implement in further projects. Thanks again, awaiting for further reviews.

Hi Sandeep, looks good in that case from my end.

**Review by hntejas — APPROVED** [review]

**Pull request merged**